src/Apps/Legion/Components/StoreFormModal/StoreFormModal.stories.tsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
```

Comment:
To avoid this, why not do Alert instead of console.log?

Reply:
we could. I figured it was less intrusive, but that would work.